app/models/location.rb

```diff
@@ -30,6 +30,14 @@ def address
     "#{street} #{house_number}, #{zip} #{city}"
   end
 
+  def calendar_event_address
```

Comment:
To ensure that this feature will work in the future when the code base changes. Could you please add a test for this?

Reply:
I added tests for this method here: https://github.com/rughh/on_ruby/pull/1036/files#diff-6b1dfa85c752473d93705b0802618ae524932f70ab05aee4f936f06886f2299dR46